stl/inc/vector

```diff
@@ -434,6 +434,40 @@ constexpr _Ty* _Unfancy_maybe_null(_Ty* _Ptr) noexcept { // do nothing for plain
     return _Ptr;
 }
 
+extern "C" {
+void _Asan_vector_aligned_annotate_start(const void* _Vector_first, const void* _Vector_last, const void* _Vector_end,
+    const size_t _Vector_new_size, const size_t _Type_size, const void** _Asan_last_out);
+void _Asan_vector_aligned_annotate_end(
+    const void* _Vector_first, const void* _Vector_last, const void* _Vector_end, const void* _Asan_last);
+void _Asan_vector_unaligned_annotate_start(const void* _Vector_first, const void* _Vector_last, const void* _Vector_end,
+    const size_t _Vector_new_size, const size_t _Type_size, const void** _Asan_last_out,
+    const void** _Aligned_first_out);
+void _Asan_vector_unaligned_annotate_end(const void* _Vector_first, const void* _Vector_last, const void* _Vector_end,
+    const void* _Asan_last, const void* _Aligned_first);
+}
+
+#ifdef __SANITIZE_ADDRESS__
+#pragma comment(lib, "stl_asan")
+#endif // __SANITIZE_ADDRESS__
+
+#pragma comment(linker, "/alternatename:_Asan_vector_aligned_annotate_start=_Asan_vector_aligned_annotate_start_noop")
+#pragma comment(linker, "/alternatename:_Asan_vector_aligned_annotate_end=_Asan_vector_aligned_annotate_end_noop")
+#pragma comment( \
+    linker, "/alternatename:_Asan_vector_unaligned_annotate_start=_Asan_vector_unaligned_annotate_start_noop")
+#pragma comment(linker, "/alternatename:_Asan_vector_unaligned_annotate_end=_Asan_vector_unaligned_annotate_end_noop")
+
+template <class _Vec, class = void>
+_INLINE_VAR constexpr bool _Has_minimum_allocation_alignment =
+    disjunction_v<bool_constant<alignof(typename _Vec::value_type) >= 8>,
+        is_same<typename _Vec::allocator_type, allocator<typename _Vec::value_type>>>;
+
+template <class _Vec>
+_INLINE_VAR constexpr bool
+    _Has_minimum_allocation_alignment<_Vec, void_t<decltype(_Vec::allocator_type::_Minimum_allocation_alignment)>> =
+        _Vec::allocator_type::_Minimum_allocation_alignment >= 8;
+
+extern bool _Asan_vector_should_annotate;

```

Comment:
Since the variable proved to be a good one, maybe it can be done `linker, "/altername..." instead of functions, then it can be `extern const`,

Reply:
Doesn't seem to make a difference in practice but I did it.